platform/darwin/src/MGLPolyline.mm

```diff
@@ -52,6 +53,27 @@ - (BOOL)isEqual:(id)other {
     return self == other || ([other isKindOfClass:[MGLPolyline class]] && [super isEqual:other]);
 }
 
+- (CLLocationCoordinate2D)coordinate {
+    mbgl::Polygon<double> polyline;
```

Comment:
@1ec5 could you please see the imp for this. I changed it according to your comment.